kit/feature/feature.go

```diff
@@ -128,6 +91,27 @@ func (*defaultFlagger) Flags(_ context.Context, flags ...Flag) (map[string]inter
 	return m, nil
 }
 
+type testingFlagger struct{}
+
+// TestingFlagger returns a flagger that always returns testing values.
+func TestingFlagger() Flagger {
+	return &testingFlagger{}
+}
+
+// Flags returns a map of testing values. It never returns an error.
+func (*testingFlagger) Flags(_ context.Context, flags ...Flag) (map[string]interface{}, error) {
+	if len(flags) == 0 {
+		flags = Flags()
+	}
```

Comment:
This isn't going to work as expected in IDPE (I've actually got an open issue because the same bug exists in the `defaultFlagger` impl above. Both of these flaggers need to accept a `[]feature.Flag` on construction, and in OSS be passed `influxdb/kit/feature.Flags()` and in IDPE be passed `idpe/kit/feature.Flags()`. This is because OSS `Flags()` in OSS returns only OSS flags, but the corresponding IDPE `Flags()` function returns the union of OSS and IDPE flags.

Basically, it just needs to be something like this 

```
type testingFlagger struct{
    flags []Flag
}

// TestingFlagger returns a flagger that always returns testing values.
func TestingFlagger(flags []Flag) Flagger {
	return &testingFlagger{
            flags: flags,
        }
}

// Flags returns a map of testing values. It never returns an error.
func (t *testingFlagger) Flags(_ context.Context, flags ...Flag) (map[string]interface{}, error) {
	if len(flags) == 0 {
		flags = t.flags
	}
 ```

If you can make the same change to the `defaultFlagger` I would appreciate it, but its less of a big deal there.